include/grpcpp/security/auth_metadata_processor.h

```diff
@@ -41,15 +43,24 @@ class AuthMetadataProcessor {
   /// a different thread from the one processing the call.
   virtual bool IsBlocking() const { return true; }
 
-  /// context is read/write: it contains the properties of the channel peer and
-  /// it is the job of the Process method to augment it with properties derived
-  /// from the passed-in auth_metadata.
-  /// consumed_auth_metadata needs to be filled with metadata that has been
-  /// consumed by the processor and will be removed from the call.
-  /// response_metadata is the metadata that will be sent as part of the
-  /// response.
-  /// If the return value is not Status::OK, the rpc call will be aborted with
-  /// the error code and error message sent back to the client.
+  /// Processes a Call associated with a Channel.
```

Comment:
Please change all occurances of "channel" to "connection" in this comment.  We do not expose "channel" as a concept on the server side.